docs/modules/servers/pages/distributed/help.adoc

```diff
@@ -0,0 +1,54 @@
+= James Distributed Server &mdash; Where to get help
+:navtitle: Help
+
+////
+
+This information is based on the email discussion from here:
+  --> https://www.mail-archive.com/server-dev@james.apache.org/msg66420.html
+
+However, I am not satisfied that I have properly captured the community's
+opinion, and there are only a few people who have contributed to the
+thread.
+
+Would be good to have some comments on this content.
+
+It could just be that I don't like the content. But if that is what the
+community thinks, then that is what we should write.
+
+////
+
+== Warning
```

Comment:
As stated in an other PR I do find this page unfair, discouraging for users , and not representative of the project reality.

I understand you want to do an electro-choc and push a SLA based vision, but writing a documentation page on an unsettled, unconsensual discussion is likely not the right way to move things forward.

Reply:
I do not disagree at all. If you see a way forward, please do propose. I was simply trying to document my understanding of what the community desires. I am also not satisfied with this content and do not want to put it up on the site, but this is all I have right now.